bin/react-scripts.js

```diff
@@ -5,14 +5,17 @@ var args = process.argv.slice(3);
 
 switch (script) {
 case 'build':
-case 'start':
 case 'eject':
+case 'start':
```

Comment:
https://github.com/facebookincubator/create-react-app/pull/250/commits/5652ad549ed6a5b56de0dad15ff1a5855d40c74c

is this what you had in mind? Because `test` doesn't have a script associated with it I wrote a quick script to add a package.json that links to the preset, runs the test and ensures a snapshot file was created.